scripts/compile-babel.js

```diff
@@ -33,7 +58,7 @@ function handleExit(code, errorCallback) {
 }
 
 function babelify(options = {}) {
-  const { watch = false, silent = true, errorCallback } = options;
+  const { watch = false, silent = false, errorCallback } = options;
```

Comment:
do you want to keep this set to true after the PR gets merged?

Reply:
good catch